plugins/arcviewer.koplugin/_meta.lua

```diff
@@ -0,0 +1,6 @@
+local _ = require("gettext")
+return {
+    name = "arcviewer",
```

Comment:
"archiveviewer" is not too long, no need to make it that concise - and "arcviewer" to me sounds like "arc format viewer".

Reply:
The plugin subdirectory should also be renamed from arcviewer.koplugin to archiveviewer.koplugin.
(Dunno if you can do that from the web interface?)